src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -353,28 +355,50 @@ internal partial class DownstreamApi : IDownstreamApi
 
             effectiveOptions.RequestAppToken = appToken;
 
-            // Obtention of the authorization header (except when calling an anonymous endpoint
-            // which is done by not specifying any scopes
-            if (effectiveOptions.Scopes != null && effectiveOptions.Scopes.Any())
-            {
+            // Obtention of the authorization header (except when calling an anonymous endpoint
+            // which is done by not specifying any scopes
+            if (effectiveOptions.Scopes != null && effectiveOptions.Scopes.Any())
+            {
                 string authorizationHeader = await _authorizationHeaderProvider.CreateAuthorizationHeaderAsync(
                        effectiveOptions.Scopes,
                        effectiveOptions,
                        user,
-                       cancellationToken).ConfigureAwait(false);
-                
+                       cancellationToken).ConfigureAwait(false);
+
                 httpRequestMessage.Headers.Add(Authorization, authorizationHeader);
-            }
-            else
-            {
-                Logger.UnauthenticatedApiCall(_logger, null);
-            }
-            if (!string.IsNullOrEmpty(effectiveOptions.AcceptHeader))
+            }
+            else
+            {
+                Logger.UnauthenticatedApiCall(_logger, null);
+            }
+            if (!string.IsNullOrEmpty(effectiveOptions.AcceptHeader))
             {
                 httpRequestMessage.Headers.Accept.ParseAdd(effectiveOptions.AcceptHeader);
-            }
-            // Opportunity to change the request message
+            }
+            // Opportunity to change the request message
             effectiveOptions.CustomizeHttpRequestMessage?.Invoke(httpRequestMessage);
-        }
-    }
-}
+        }
+
+
+        private static readonly Dictionary<string, string> s_callerSDKDetails = new()
+          {
+              { "caller-sdk-id", "1" },  // 1 = Downstream API SDK ID

```

Comment:
is there a doc on this?

Reply:
I'm not following how this will impact our current server telemetry and reporting.

Reply:
The caller sdk api id and version will be sent to the server side telemetry from MSAL.Net as a platform field. Currently we have a [PR](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/nebharg/ztaSupportSdkInfo/src/client/Microsoft.Identity.Client/TelemetryCore/Http/HttpTelemetryManager.cs#L23-L24) for this